frontend/src/mocks/handlers.ts

```diff
@@ -25,6 +26,19 @@ export const toPaginatedResponse = (results: any[]): typeof EMPTY_PAGINATED_RESP
     previous: null,
 })
 
+// this really returns MaybePromise<ResponseFunction<any>>
+// but MSW doesn't export MaybePromise 🤷
+function posthogCORSResponse(req: RestRequest, res: ResponseComposition, ctx: RestContext): any {
+    return res(
+        ctx.status(200),
+        ctx.json('ok'),
+        // some of our tests try to make requests via posthog-js e.g. userLogic calls identify
+        // they have to have CORS allowed, or they pass but print noise to the console
+        ctx.set('Access-Control-Allow-Origin', req.referrer.length ? req.referrer : 'http://localhost'),
+        ctx.set('Access-Control-Allow-Credentials', 'true')
```

Comment:
maybe we should stop the calls to posthog
but someone would accidentally introduce them somewhere else